Documentation/distributions.md

```diff
@@ -0,0 +1,46 @@
+# How to get rkt on each Linux distribution
+
+## CoreOS
+
+rkt is pre-installed on CoreOS.
+
+## Fedora
+
+rkt is packaged in Fedora Rawhide:
+```
+sudo dnf install rkt
+```
+
+Other versions of Fedora can get [the last release](https://github.com/coreos/rkt/releases)
+
+#### Caveat: SELinux
+
+rkt currently does not integrate with SELinux on Fedora.
+See [#1727](https://github.com/coreos/rkt/issues/1727).
+As a workaround, SELinux can be temporarily disabled:
+```
+sudo setenforce Permissive
+```
+Or permanently disabled by editing `/etc/selinux/config`:
+```
+SELINUX=permissive
+```
+
+#### Caveat: firewall
+
+The default firewall rules can block the traffic from rkt pods.
+As a workaround, they can be removed:
+```
+sudo iptables -F
+sudo iptables -F -t nat
+```
+
+## Arch
+
+rkt is packaged in Arch.
+```
+pacman -S rkt
+```
```

Comment:
It is not in the official repositories, but in [AUR](https://wiki.archlinux.org/index.php/Arch_User_Repository): https://aur.archlinux.org/packages/rkt

So to install it you need to follow the instructions [here](https://wiki.archlinux.org/index.php/Arch_User_Repository#Installing_packages) or use a [AUR helper](https://wiki.archlinux.org/index.php/AUR_helpers). I use packer.